### PR TITLE
[tests-only] change the file content of a received share

### PR DESCRIPTION
**Pull request by ScharfViktor**
## Describe the bug
A test is written that demonstrates the error. The test will work when the bug is fixed

## Steps to reproduce

User1 creates file and shares file to User2
User2 accepts file and try to upload file with some name
Or run test: yarn run test:acceptance:ocis tests/acceptance/features/webUIUserJourney/journey1.feature:29

## Expected behavior
Created new version

## Actual behavior
403 Forbinden

125840261-52e59662-0d43-4672-9dac-0c3b3cb9e72e

## Setup
Please describe how you started the server and provide a list of relevant environment variables.

OCIS_VERSION=v1.9.0 reachable at https://localhost:9200
BRANCH=v1.9.0-rc1

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalUsers</strong> https://drone.owncloud.com/owncloud/web/17534/24/1
<details><summary>:boom: The acceptance tests failed. Please find the screenshots inside ...</summary>

<p>


### webUISharingInternalUsersShareWithPage-shareWithUsers-feature-91.png
  


</p></details>
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalGroupsSharingIndicator</strong> https://drone.owncloud.com/owncloud/web/17536/23/1
The following scenarios passed on retry:
- webUISharingInternalGroupsSharingIndicator/shareWithGroups.feature:114

File: tests/acceptance/features/webUIFilesActionMenu/versions.feature [hunk 1]
```diff
@@ -83,3 +83,13 @@ Feature: Versions of a file
     When the user re-logs in as "Alice" using the webUI
     And the user browses to display the "versions" details of file "lorem-file.txt"
     Then the versions list should contain 0 entries
+
```
Comment (phil-davis) [inline tests/acceptance/features/webUIFilesActionMenu/versions.feature:86]:
```suggestion

  @issue-5534
```

Actually #5534 is a closed PR. This needs to point to an issue.

File: tests/acceptance/expected-failures-with-oc10-server-oauth2-login.md [hunk 2]
```diff
@@ -164,4 +164,7 @@ Other free text and markdown formatting can be used elsewhere in the document if
 -   [webUIFiles/browseDirectlyToDetailsTab.feature:42](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIFiles/browseDirectlyToDetailsTab.feature#L42)
 
 ### [reshared share that is shared with a group the sharer is part of shows twice on "Share with me" page](https://github.com/owncloud/web/issues/2512)
--   [webUISharingAcceptShares/acceptShares.feature:31](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingAcceptShares/acceptShares.feature#L31)
\ No newline at end of file
+-   [webUISharingAcceptShares/acceptShares.feature:31](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingAcceptShares/acceptShares.feature#L31)
+
+### [Change the file content of a received shared file](https://github.com/owncloud/web/pull/5534)
```
Comment (phil-davis) [inline tests/acceptance/expected-failures-with-oc10-server-oauth2-login.md:169]: PR #5534 is closed. Is there an issue for this?
The links in expected-failures should point to an issue.

File: tests/acceptance/expected-failures-with-ocis-server-ocis-storage.md [hunk 3]
```diff
@@ -158,6 +158,9 @@ Other free text and markdown formatting can be used elsewhere in the document if
 -   [webUIFilesActionMenu/versions.feature:59](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIFilesActionMenu/versions.feature#L59)
 -   [webUIFilesActionMenu/versions.feature:74](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIFilesActionMenu/versions.feature#L74)
 
+### [Change the file content of a received shared file](https://github.com/owncloud/web/pull/5534)
```
Comment (phil-davis) [inline tests/acceptance/expected-failures-with-ocis-server-ocis-storage.md:161]: PR #5534 is closed. Is there an issue for this?
The links in expected-failures should point to an issue.

File: tests/acceptance/features/webUIFilesActionMenu/versions.feature (hunk 1 above)
Comment (ScharfViktor) [reply tests/acceptance/features/webUIFilesActionMenu/versions.feature:86]: It's not issue. It's PR. Maybe something like @web-pull-5548?

**Comment by ownclouders** [comment]
Results for <strong>oC10XGAPortrait2</strong> https://drone.owncloud.com/owncloud/web/17572/42/1
The following scenarios passed on retry:
- webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature:34
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalUsersSharingIndicator</strong> https://drone.owncloud.com/owncloud/web/17579/27/1
The following scenarios passed on retry:
- webUISharingInternalUsersToRootSharingIndicator/shareWithUsers.feature:98

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingExternalRoot</strong> https://drone.owncloud.com/owncloud/web/17579/39/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingAccept</strong> https://drone.owncloud.com/owncloud/web/17581/10/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oC10CreateDelete</strong> https://drone.owncloud.com/owncloud/web/17581/8/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

File: tests/acceptance/features/webUIFilesActionMenu/versions.feature (hunk 1 above)
Comment (phil-davis) [reply tests/acceptance/features/webUIFilesActionMenu/versions.feature:86]: The PR will be in closed/merged state when it is approved and merged. So developers will not notice it when scheduling work in sprints. We should raise an issue that describes the problem, and link to the issue number here and in the expected-failures files.

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/17582/49/1
The following scenarios passed on retry:
- webUISharingAcceptShares/acceptShares.feature:109

File: tests/acceptance/features/webUIFilesActionMenu/versions.feature (hunk 1 above)
Comment (ScharfViktor) [reply tests/acceptance/features/webUIFilesActionMenu/versions.feature:86]: A mistake on the backend side. I created issue in ocis and added points to an issue

**Review by phil-davis — APPROVED** [review]

**Pull request merged**